### PR TITLE
Add thanks page in the site

### PR DESCRIPTION
**Pull request by aduprat**
james_site_thanks

### CODE REVIEW

File: src/site/xdoc/thanks.xml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    <table>
+
+        <tr>
+          <th width="30%">Company</th>
+          <th>Description</th>
+        </tr>
+
+        <tr>
+          <td><img src="images/linagora.png" /></td>
+          <td><p>We are an Open Source Software Company.</p>
```
Comment (fvn-linagora) [inline src/site/xdoc/thanks.xml]: "OSS company" is a broad term.
Being referred to as an ISV doing only OSS projects might be closer to what linagora actually is ?
We might also include references to products we're "shipping" (OBM and co.), if appropriate.

**Comment by rouazana** [comment]
Here are some samples of professional support pages (generally linked from contact page and/or faq):
http://camel.apache.org/commercial-camel-offerings.html
http://activemq.apache.org/support.html
https://mahout.apache.org/general/professional-support.html

File: src/site/xdoc/thanks.xml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+              <p>Our mission is to bring digital independency to our customers.</p>
```
Comment (fvn-linagora) [inline src/site/xdoc/thanks.xml]: I guess digital independence sounds more english.

File: src/site/xdoc/thanks.xml [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    <subsection name="Companies description">
+    <table>
+
+        <tr>
+          <th width="30%">Company</th>
+          <th>Description</th>
+        </tr>
+
+        <tr>
+          <td><img src="images/linagora.png" /></td>
```
Comment (aduprat) [inline src/site/xdoc/thanks.xml]: The logo should link to Linagora site.

**Comment by aduprat** [comment]
test this please

**Comment by aduprat** [comment]
test this please

File: src/site/xdoc/thanks.xml (hunk 1 above)
Comment (aduprat) [inline src/site/xdoc/thanks.xml]: Copied from Linagora site, will keep this to stay consistent.

File: src/site/xdoc/thanks.xml (hunk 2 above)
Comment (aduprat) [inline src/site/xdoc/thanks.xml]: same comment

File: src/site/xdoc/thanks.xml (hunk 3 above)
Comment (aduprat) [inline src/site/xdoc/thanks.xml]: OK

**Comment by aduprat** [comment]
james_site_thanks

**Comment by mbaechler** [comment]
I think it's not ok : Linagora only contributes "Jenkins CI".

The sentence "Linagora employs 4 James commiters and PMC members. It can offer professional services related to James : support, feature development or project integration." should be in company description.

Finally, Linagora contributions should only be Jenkins CI.

**Comment by aduprat** [comment]
capture du 2016-02-25 14 33 42

**Comment by aduprat** [comment]
capture du 2016-02-26 10 29 10
capture du 2016-02-26 10 28 50

**Comment by rouazana** [comment]
:+1:

File: src/site/xdoc/support.xml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  <section name="Professional support">
+
+    <p>Here is a list of companies which can provide you some support on James.</p>
+
+    <subsection name="Linagora">
+      <a href="http://www.linagora.com/"><img src="images/linagora.png" /></a>
+      <p>We are an Open Source Software Company.</p>
+      <p>Our mission is to bring digital independency to our customers.</p>
+      <p>We are focused on large organizations both from private and public sector. We have set ourselves the goal of helping Governments to achieve their digital sovereignty.</p>
+      <p>Linagora employs 4 James commiters and PMC members. We can offer professional services related to James: support, feature development or project integration.</p>
```
Comment (mbaechler) [inline src/site/xdoc/support.xml]: 3 PMC members

File: src/site/xdoc/thanks.xml [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+          <th width="30%">Company</th>
+          <th>Description</th>
+        </tr>
+
+        <tr>
+          <td><a href="http://www.linagora.com/"><img src="images/linagora.png" /></a></td>
+          <td><p>We are an Open Source Software Company.</p>
+              <p>Our mission is to bring digital independency to our customers.</p>
+              <p>We are focused on large organizations both from private and public sector. We have set ourselves the goal of helping Governments to achieve their digital sovereignty.</p>
+              <p>Linagora employs 4 James commiters and PMC members. We can offer professional services related to James: support, feature development or project integration.</p>
```
Comment (mbaechler) [inline src/site/xdoc/thanks.xml]: 3 PMC members

File: src/site/xdoc/support.xml (hunk 4 above)
Comment (aduprat) [inline src/site/xdoc/support.xml]: OK

File: src/site/xdoc/thanks.xml (hunk 5 above)
Comment (aduprat) [inline src/site/xdoc/thanks.xml]: OK

**Comment by MichaelBailly** [comment]
Are we allowed to put a link on the words "professional services" ? If yes, here it is: 

https://www.linagora.com/ossa

**Comment by aduprat** [comment]
Well, not on "professional services" i think; but i upgraded the Linagora link on this page.

**Comment by MichaelBailly** [comment]
:+1:

**Comment by mbaechler** [comment]
:+1:

**Pull request merged**